src/main/java/emstask/spring/controller/EmployeeController.java

```diff
@@ -0,0 +1,52 @@
+package emstask.spring.controller;
+import emstask.spring.dao.DesignationRepository;
+import emstask.spring.dao.EmployeeRepository;
+import emstask.spring.model.EmployeePost;
+import emstask.spring.model.EmployeePut;
+import emstask.spring.service.EmployeeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/rest")
+public class EmployeeController
+{
+    @Autowired
+    EmployeeRepository empRepo;
+    @Autowired
+    DesignationRepository degRepo;
+    @Autowired
+    EmployeeService empService;
+
+    @GetMapping(path = "/employees",produces = "application/json")
+    public ResponseEntity getAllEmployees()
+    {
+       return empService.getAll();
+    }
+
+    @PostMapping(path = "/employees")
+    public ResponseEntity createEmployee(@RequestBody EmployeePost employee)
+    {
+         return empService.addUser(employee);

```

Comment:
Usually responseentity objects are created in controllers and will be using the object returned by a service. In your case service creates resp entity objects.